src/source/vector_tile_worker_source.js

```diff
@@ -214,7 +223,7 @@ class VectorTileWorkerSource extends Evented implements WorkerSource {
                     if (err || !result) return callback(err);
 
                     // Transferring a copy of rawTileData because the worker needs to retain its copy.
-                    callback(null, extend({rawTileData: rawTileData.slice(0)}, result, cacheControl, resourceTiming));
+                    callback(null, extend({rawTileData: rawTileData.slice(0)}, result, cacheControl, this.deduped._resourceTiming));
```

Comment:
Could we get rid of the `RequestPerformance` class and replace it with a method that does what `getMeasurement()` does?

Both here and in geojson_worker_source.js we could do something like this:

`_resourceTiming: getResourceTiming(params.request)`

and remove the:
- new RequestPerformance()
- the stuff in DedupedRequest

Reply:
Yep 👍 much simpler